### ISSUE TITLE
13 Hazards Parameter

### ISSUE THREAD

**Issue by fadmastuti** [urbanriskmap/cognicity-grasp#7]
BNPB released regulation (no 2 year 2012) to standardise Indonesian hazards parameter. Please look into this [link](http://www.gitews.org/tsunami-kit/en/E6/further_resources/national_level/peraturan_kepala_BNPB/Perka%20BNPB%202-2012_Pedoman%20Umum%20Pengkajian%20Risiko%20Bencana.PDF) to see the original document.

_*start from page 15 - table 1 on the document_

**Finding:**

![img_20161012_150637](https://cloud.githubusercontent.com/assets/2441950/19302771/d7329896-908f-11e6-8274-1df19aa2c0bc.jpg)

Basically, the alert levels for all hazards described as:
1. low (green)
2. medium (yellow)
3. tinggi (high)

The thresholds for each level in each hazards are referring to the national standards that are released by related agency (i.e extreme weather by BMKG, etc).
We will follow up to BNPB next week if there is any update on this regulation while we have a discussion with PDC as well.

**cc** @talltom @anexact @prittandrani @nashinmahtani @clgeros

**Comment by talltom** [comment]
Hi @fadmastuti + @anexact  - please can you also inquire if there is standard for duration of alerts?

I know that BPBD have requested this be added to flood affected areas, so it would be good to get input from BNBP and PDC to see what they do currently and try and maintain interoperability with their processes.

**Comment by fadmastuti** [comment]
@talltom Yes. I am currently looking into PDC concept of operations documentation for InAWARE and DisasterAWARE, trying to find the thresholds and duration for the alerts. We will also follow up to BNPB as I found an un-match parameter between BNPB standard and PDC doc (esp. on flooding).

We will update to you as soon as we got the parameter cleared.

**Comment by fadmastuti** [comment]
After looked at the DisasterAWARE and InAWARE standards parameter, here is the conclusion:

**DisasterAWARE hazard durations standard:**
_*an early warning and decision support platform by PDC (see this [link](http://www.pdc.org/solutions/products/disasteraware/) for detail explanation)._

| No  | Hazards                        | Blue                  | Green                 | Yellow                 | Red                     |
| ---- | -------------------------  | ------------------ | ------------------ | ------------------- | ------------------- |
| 1     | Flood                            | 1                       | 1                         | 2                         | 2-5                      | 
| 2     | Drought                        | -                        | 3                         | 3                         | 5                         |
| 3     | Landslide                     | 1-2                    | 1                         | 2                         | 3                         |
| 4     | Volcanoes                    | 2                        | 7                        | 3                          | 4-5                     |
| 5     | Wildfire                         | 1                        | 1                        | 2                          | 3-5                     |
| 6     | Extreme temperature   | 1                        | 1                        | 2                          | 3-5                     |
| 7     | High wind                     | -                        | -                         | 1                          | 2                        |
| 8     | Manmade                     | 1                       | 1                         | 2                          | 2                        |
| 9     | Marine                          | 1                       | 1                         | 2                          | 3                        |
| 10   | Storm                           | 1-2                    | 1                         | 2                          | 3                        |
| 11   | Tornado                        | 1-3                    | 1                         | 2                          | 3                        |
| 12   | Winter storm                | -                         | 1                         | 2                         | 3                        |
| 13   | Avalance                      | 1                        | 1                         | 2                          | 3                        |
| 14   | Biomedical                   | 1 month            | 2 months            | 3 months             | 3-6 months        |

_*numerous values in days except biomedical hazard_
 

**InAWARE documentation only mention 8 Hazards and its alert thresholds:**

I only find 8 Hazards on InAWARE concepts of operation documentation. There are:
1. Earthquake
2. Tsunami
3. Volcanic
4. Tropical cyclone
5. Wildfire
6. Flood
7. Drought
8. Landslide

It also mentioned the thresholds for each hazard, but let's focus on the Flood hazard first.

The Flood determined as four stages:

| Hazard level                        | Parameter (flood height)                        | Colours*                       | 
| ------------------------------  | -------------------------------------------- | ------------------------- |
| Major                                   | > 200 cm                                               | Red                              | 
| Warning                               | 150 - 200 cm                                         | Yellow                          | 
| Alert                                     | 80 - 150                                                 | Green                          | 
| Prepared                              | 50 - 80                                                   | Blue                            | 

_*the colours is not confirm yet_

However, none of the flood threshold standard version is match (BPBD DKI vs PDC vs BNPB). We are about meeting PDC team (incl. PDC's Geospatial Specialist) next Monday and have everyone on the same table on Tuesday. We will follow up this issue on the meeting next week and report back to you as soon as we solve this @talltom  .

cc @anexact @prittandrani
